### PR TITLE
feat(DTFS2-7087): add audit logs to create user endpoint

### PR DESCRIPTION
**Pull request by MarRobSoftwire**
## Introduction :pencil2:
All endpoints in dtfs-central-api should add `auditDetails` when inserting/updating data. This PR covers the create user endpoint

## Resolution :heavy_check_mark:
- Add audit details to database changes for create user endpoint

## Note ⚠️ 
AFAIK this endpoint is not used anywhere, either this should be deleted, or the tfm-api create users endpoint call this instead

### CODE REVIEW

File: dtfs-central-api/src/v1/routes/tfm-routes.js [hunk 1]
```diff
@@ -671,6 +671,14 @@ tfmRouter.route('/users').get(
  *         application/json:
  *           schema:
  *             $ref: '#/definitions/TFMUser'
+ *             auditDetails:
```
Comment (ChristianMcc) [inline dtfs-central-api/src/v1/routes/tfm-routes.js:674]: One to think about, if this is the same everywhere we might want to make a definition of it and just reference that everywhere

**Review by ChristianMcc — APPROVED** [review]
LGTM :) one thing to think about for these PRs

File: dtfs-central-api/src/v1/routes/tfm-routes.js (hunk 1 above)
Comment (MarRobSoftwire) [reply dtfs-central-api/src/v1/routes/tfm-routes.js:674]: Its nearly the same, just has different enum values - not sure if there's a nice way to handle this without repetition

Comment (MarRobSoftwire) [reply dtfs-central-api/src/v1/routes/tfm-routes.js:674]: Have decided to just do this in a verbose way as I think its neater - added to https://github.com/UK-Export-Finance/dtfs2/pull/2948 though, not here

**Pull request merged**